### PR TITLE
Module_Cms MFTF test improvements

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
### Description (*)

1. AdminCreateCmsPageTest doesn't have delete action, after creating a CMS page. Added a CMS page delete action in the existing test.
2. A single test file contains multiple tests. So moved the test to separate files to follow best practices.

### Manual testing scenarios (*)

1. Run the AdminCreateCmsPageTest in the local environment and verified test is running.
2. Run the AdminCreateDuplicatedCmsPageTest in the local environment and verified test is running.
3. Run the AdminConfigDefaultCMSPageLayoutFromConfigurationSettingTest in the local environment and verified test is running.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by rogyar** [comment]
Hi @ajithkumar-maragathavel. Thank you for your collaboration. In your next PRs you can also consider rebuilding the tests according to the [best practices](https://github.com/magento/magento-functional-tests-migration/wiki/Best-practices). For example, building a test body using action groups only. But keep backward compatibility in mind.

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-6838](https://jira.corp.magento.com/browse/ENGCOM-6838) has been created to process this Pull Request

**Comment by ajithkumar-maragathavel** [comment]
@rogyar I will update the test using only the action group in my next PR. Thanks for the review and providing the best practices link.

File: app/code/Magento/Cms/Test/Mftf/Test/AdminCreateDuplicatedCmsPageTest.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            <stories value="CMS Page Duplication and Reset Removal MAGETWO-87096"/>
+            <title value="Admin should be able to duplicate a CMS Page"/>
+            <description value="Admin should be able to duplicate a CMS Page"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MAGETWO-89184"/>
+            <group value="Cms"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginActionGroup" stepKey="loginGetFromGeneralFile"/>
+            <actionGroup ref="DisabledWYSIWYGActionGroup" stepKey="disableWYSIWYG"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Cms/Test/Mftf/Test/AdminCreateDuplicatedCmsPageTest.xml:23]: @rogyar I don't remember, but it's no longer necessary to add this. How do you think?

**Review by lbajsarowicz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6838](https://jira.corp.magento.com/browse/ENGCOM-6838) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Pull request merged**